#!/usr/bin/python3

def raise_exception_msg(message=""):
    # Raise a nameError with a custom message
    raise NameError(message)
